### PR TITLE
Added app id to the metrics

### PR DESCRIPTION
**Pull request by ravitb**
#### Main purpose:
We want to be able to analyze the data in our metrics system by app id. This information was added to the sent requests.
 
#### Technical description:
Added app_id to each metric

### CODE REVIEW

File: scripts/src/utils.ts [hunk 1]
```diff
@@ -2,6 +2,12 @@ import * as _path from "path";
 import { join } from "path";
 import * as fs from "fs";
 
+import { Express } from "express";
+import { Context } from "./public/routes";
+export interface RequestWithContext extends Express.Request {
+	context?: Context;
```
Comment (nitzantomer) [inline scripts/src/utils.ts]: where are you setting this?

File: package.json [hunk 2]
```diff
@@ -11,55 +11,55 @@
...
-    "@types/sequelize": "^4.27.4",
-    "@types/winston": "^2.3.7",
-    "npm-run-all": "^4.0.1",
+    "@types/body-parser": "^1.17.0",
+    "@types/express": "^4.16.0",
+    "@types/jsonwebtoken": "^7.2.8",
+    "@types/redlock": "^3.0.2",
+    "@types/sequelize": "^4.27.33",
+    "@types/winston": "^2.4.4",
+    "npm-run-all": "^4.1.5",
```
Comment (yosriz) [inline package.json]: why did we update versions?

File: scripts/src/utils.ts (hunk 1 above)
Comment (ravitb) [reply scripts/src/utils.ts]: This code is already exist, I am just using it here as well.

File: package.json (hunk 2 above)
Comment (ravitb) [reply package.json]: Because some of the modules are out of date.

File: scripts/src/utils.ts (hunk 1 above)
Comment (nitzantomer) [reply scripts/src/utils.ts]: it's probably not the right place to put it then

**Pull request merged**